docs/server/server.cfg.md

```diff
@@ -3,6 +3,8 @@ title: "server.cfg"
 description: Server configuration file.
 ---
 
+# config.json is now used on open.mp instead of server.cfg! 
```

Comment:
this would be nicer as a note, like these:

![image](https://user-images.githubusercontent.com/1636971/221160626-fc40e61e-1cfb-4789-8f4c-5cc27d16036d.png)


It's possible with the following syntax:

```suggestion
:::note

config.json is now used on open.mp instead of server.cfg

:::
```